handlers/gke.go

```diff
@@ -26,7 +33,39 @@ func findUnusedGKECluster(clusterMap *clusters.Map, leaseMap *leases.Map) (*cont
 	return nil, errUnusedGKEClusterNotFound
 }
 
-func createKubeConfigFromCluster(cluster *container.Cluster) string {
-	// TODO: implement
-	return ""
+func createKubeConfigFromCluster(cluster *container.Cluster) ([]byte, error) {
+	contextName := strings.ToLower(cluster.Name)
+	authInfoName := strings.ToLower(cluster.Name)
```

Comment:
Fixed in https://github.com/deis/k8s-claimer/pull/24/commits/f5dfb9697217ccb90d764c7ff91867855ea32b16